addons/crm/tests/test_crm_pls.py

```diff
@@ -399,3 +399,17 @@ def test_settings_pls_start_date(self):
         res_config_new = resConfig.new()
         self.assertEqual(fields.Date.to_string(res_config_new.predictive_lead_scoring_start_date),
             str_date_8_days_ago, "If config param is not a valid date, date in settings should be set to 8 days before today")
+
+    def test_stages_team_specific(self):
+        Lead = self.env['crm.lead']
+        state_value = 'correct'
+        source_id = self.env['utm.source'].search([], limit=1).id
```

Comment:
Do source_id really have any influence here ? Purpose would be to limit test variables to what is really necessary for our test. Otherwise let us remove noise in the test and keep only relevant stuff.